packages/edit-site/src/components/dataviews/view-table.js

```diff
@@ -274,7 +342,15 @@ function ViewTable( {
 		}
 
 		return _columns;
-	}, [ fields, actions, view ] );
+	}, [
+		areAllSelected,
```

Comment:
Is this necessary? It depends on `selection` changes, so do we have every case covered with that dependency? Anyway, if we move the line where `areAllSelected` inside the `useMemo` we could remove it.